src/utils/optionsparser.cc

```
@@ -50,12 +50,14 @@ std::vector<std::string> OptionsParser::ListOptionsUci() const {
   return result;
 }
 
-void OptionsParser::SetOption(const std::string& name, const std::string& value,
+bool OptionsParser::SetOption(const std::string& name, const std::string& value,
                               const std::string& context) {
   auto option = FindOptionByName(name);
   if (option) {
     option->SetValue(value, GetMutableOptions(context));
+    return true;
```

Maybe also just throw Exception rather than return `false`? :)